_test/test_dnd_class/test_dnd_constructor.m

```diff
@@ -65,8 +121,27 @@ function test_d2d_contains_expected_properties(obj)
         end
     end
 
+    function test_d0d_get_returns_set_properties(obj)
+        dnd_obj = d0d();
+        obj.assert_dnd_get_returns_set_properties(dnd_obj);
+    end
+    function test_d1d_get_returns_set_properties(obj)
```

Comment:
Is the intent of not having whitespace between these tests to group them? Would a comment, or even a section delimiter (`%%`), be a better way of grouping them? If we ever decide on a formatter this grouping will be lost